test/controllers/draft_test.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+const chai = require('chai');
+chai.use(require('chai-http'));
+chai.use(require('chai-subset'));
+
+const server = require('../../app/index');
+const knex    = require('../../app/lib/bookshelf').knex;
+const Draft= require('../../app/models/draft');
+
+const testdata = require('../../seeds/test_data.json');
+
+describe.only('Draft Controller', function() {
+
```

Comment:
Your tests will all need to create JWTs and add them as authentication headers. Look at the `user` tests for an example of how to do this.